packages/docs-reanimated/src/theme/DocSidebar/index.js

```diff
@@ -15,11 +15,13 @@ export default function DocSidebarWrapper(props) {
 
   const newItems = ['animations/withClamp'];
   const experimentalItems = ['shared-element-transitions/overview'];
+  const unreleasedItems = ['debugging/accurate-call-stacks', 'debugging/logger-configuration'];
```

Comment:
Unfortunately, it is not possible to create a single label near the group title. We can only add labels to specific pages links in the sidebar.